extmod/modtls_mbedtls.c

```diff
@@ -37,6 +37,8 @@
 #include "py/stream.h"
 #include "py/objstr.h"
 #include "py/reader.h"
+#include "py/smallint.h"
```

Comment:
This header shouldn't be needed anymore.

Reply:
Removed!